integration/secret/secret_test.go

```diff
@@ -41,3 +43,110 @@ func TestSecretInspect(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, secret.ID, secretResp.ID)
 }
+
+func TestSecretList(t *testing.T) {
+	skip.If(t, testEnv.DaemonInfo.OSType != "linux")
+
+	defer setupTest(t)()
+	d := swarm.NewSwarm(t, testEnv)
+	defer d.Stop(t)
+	client, err := request.NewClientForHost(d.Sock())
+	require.NoError(t, err)
+
+	ctx := context.Background()
+
+	testName0 := "test0"
+	testName1 := "test1"
+
+	// create secret test0
+	secret0, err := client.SecretCreate(ctx, swarmtypes.SecretSpec{
```

Comment:
Maybe a helper function that creates a secret with a name and a labels, and returns the id? 

```go
secret0 := createSecret(t, secretName0, map[stringstring{"type": "test"})
secret1 := createSecret(t, secretName1, map[stringstring{"type": "production"})
```